icons/figma/figma-original.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 300"><style>.st0{fill:#0acf83}.st1{fill:#a259ff}.st2{fill:#f24e1e}.st3{fill:#ff7262}.st4{fill:#1abcfe}</style><path id="path0_fill" class="st0" d="M50 300c27.6 0 50-22.4 50-50v-50H50c-27.6 0-50 22.4-50 50s22.4 50 50 50z"/><path id="path1_fill" class="st1" d="M0 150c0-27.6 22.4-50 50-50h50v100H50c-27.6 0-50-22.4-50-50z"/><path id="path1_fill_1_" class="st2" d="M0 50C0 22.4 22.4 0 50 0h50v100H50C22.4 100 0 77.6 0 50z"/><path id="path2_fill" class="st3" d="M100 0h50c27.6 0 50 22.4 50 50s-22.4 50-50 50h-50V0z"/><path id="path3_fill" class="st4" d="M200 150c0 27.6-22.4 50-50 50s-50-22.4-50-50 22.4-50 50-50 50 22.4 50 50z"/></svg>
```

Comment:
[CONTRIBUTING.md#organizational-guidelines](https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#organizational-guidelines): .svg file contains one version of an icon in a "0 0 128 128" viewbox

Reply:
![screencapture-file-C-Users-HP-Documents-Github-devicon-icons-figma-figma-original-svg-2020-07-29-16_48_22](https://user-images.githubusercontent.com/56074407/88794330-efc55d00-d1bb-11ea-9985-8c20a158a8d8.png)

This is how my Chrome window (1440x1080) is displaying the Figma svg and if I set the viewBox to 0 0 128 128 it's getting cropped out and not displaing correctly. I can't figure out why. I tried to do `preserveAspectRatio="xMidYMid meet"` in the second case but that didn't change anything.

![screencapture-file-C-Users-HP-Documents-Github-devicon-icons-figma-figma-original-svg-2020-07-29-16_55_14 (1)](https://user-images.githubusercontent.com/56074407/88794678-74b07680-d1bc-11ea-95e4-02eaf02eb9af.png)